astacus/common/ipc.py

```diff
@@ -256,7 +256,9 @@ class ListSingleBackup(AstacusModel):
     attempt: int
     nodes: int
     files: int
+    cluster_files: int
     total_size: int
+    cluster_data_size: int
     upload_size: int
     upload_stored_size: int
```

Comment:
It would be good to document here (with comments) what Kevin noted in the PR, e.g. what each of these means. Also fields from nodes onward are actually NOT from BackupManifest so they should be really documented here.

Reply:
Points adressed